unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/WearableItem.cs

```diff
@@ -182,6 +182,16 @@ public static HashSet<string> CompoundHidesList(string bodyShapeId, List<Wearabl
 
         return result;
     }
+
+    //Workaround to know the net of a wearable. 
+    //Once wearables are allowed to be moved from Ethereum to Polygon this method wont be reliable anymore
+    //To retrieve this properly first we need the catalyst to send the net of each wearable, not just the ID
+    public bool IsInL2()
+    {
+        if (id.StartsWith("urn:decentraland:matic") || id.StartsWith("urn:decentraland:mumbai"))
```

Comment:
Only as suggestion: As we have to check specifics texts to know if a wearable is in L2, I would move these 2 strings to constants in order to make them a bit more trackeable.

Reply:
I intended to leave them as literals here since this is completely something temporary until catalysts provide the wearable network. I want to encapsulate the workaround as much as possible.

Reply:
Ok, if it is something temporal, no problem 👍